_data/hosting_providers.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+    "name": "Pantheon",
+    "link": "https://pantheon.io/",
+    "category": "full",
+    "tutorial": "https://pantheon.io/docs/guides/launch/",
+    "announcement": "https://pantheon.io/blog/pantheon-launches-global-cdn-automated-https-all-sites",
+    "plan": "",
+    "reviewed": "",
+    "note": "Website Ops for Drupal & WordPress"
```

Comment:
as per the comment above, are there some hosting products you offer that have https set up automatically and others that don't? this is a great field to specify which products offer completely automatic https setup.

Reply:
Thanks for the review @ohemorange!

 All Pantheon sites include automated HTTPS so I removed the note and tutorial link.